packages/manager/modules/pci/src/projects/project/storages/containers/container/translations/Messages_fr_FR.json

```diff
@@ -7,8 +7,8 @@
   "pci_projects_project_storages_containers_container_info_publicUrl": "Endpoint",
   "pci_projects_project_storages_containers_container_info_publicUrl_help": "Cette information est nécessaire pour récupérer ou manipuler vos objets via l'API. Rendez-vous dans la section guide pour plus de détails.",
   "pci_projects_project_storages_containers_container_archive_info_staticUrl": "Adresse du serveur pour rsync/sftp/scp :",
-  "pci_projects_project_storages_containers_container_object_info_staticUrl": "Valeur de l'enregistrement CNAME/TXT :",
-  "pci_projects_project_storages_containers_container_object_info_staticUrl_help": "Utilisez le conteneur \"Hébergement statique\" sur un domaine ou sous-domaine. Modifiez simplement la zone DNS en ajoutant un enregistrement CNAME avec cette valeur.",
+  "pci_projects_project_storages_containers_container_object_info_staticUrl": "Virtual Hosted-Style :",
```

Comment:
**question:** Do we need translation for this? Looks like a constant.

Reply:
For me it's a content.
Even it's a Globish phrase we have to put it into content.

Why: because we can easily adapt the content if the name differ for others language or we have to change the content again